app/assets/javascripts/tribo_viva/checkouts/checkout.js.coffee

```diff
@@ -0,0 +1,97 @@
+TriboViva.Checkouts ?= {}
+
+TriboViva.Checkouts.Checkout =
+  init: ->
+    Iugu.setAccountID("79d7660c-6169-4e07-aed9-8a0883d5249a")
+    Iugu.setTestMode(true) # disable on production
+
+    # Set mask for card expiration date
+    $(".js-credit-card-number").mask("9999 9999 9999 9999")
```

Comment:
eu tiraria o jquery mask que tem hoje no app e usaria o [Formatter.js](https://github.com/firstopinion/formatter.js) em todo ele (não só no pagamento). Se tu for ver o código do [iugu.js](https://js.iugu.com/v2) descompactado, vai ver que ele vê se tem o formatter e se tiver, já faz toda a mão das máscaras automaticamente de acordo com o cartão que tu digita. colocando uma classe da bandeira no form e ajustando automaticamente a máscara pro tipo do cartão. visa e master tem 16 dígitos, funcionaria com essa máscara de hoje. mas amex por exemplo tem 15 dígitos, o mask fixo desse jeito nunca ia deixar o usuário digitar o cartão